sdk/core/azure-core-test/src/main/java/com/azure/core/test/models/RecordFilePayload.java

```diff
@@ -16,12 +16,20 @@ public class RecordFilePayload {
     @JsonProperty("x-recording-file")
     private String recordingFile;
 
+    /**
+     * The asset file path
+     */
+    @JsonProperty("x-recording-assets-file")
+    private String assetFile;
+
     /**
      * Creates an instance of {@link RecordFilePayload}.
      * @param recordingFile The partial path to the recording file.
+     * @param assetFile The path to asset file.
      */
-    public RecordFilePayload(String recordingFile) {
+    public RecordFilePayload(String recordingFile, String assetFile) {
```

Comment:
Don't we need `JsonCreator` here? Or does this only get serialized? If this class is serialize only could we add `WRITE_ONLY` to the JSON properties